python/nav/alertengine/dispatchers/jabber_dispatcher.py

```diff
@@ -16,18 +16,24 @@
 #
 """Plugin module for sending jabber alerts"""
 
-import xmpp
 import logging
 import time
 from threading import Thread
 from time import sleep
 
 from nav.errors import ConfigurationError
 from nav.alertengine.dispatchers import Dispatcher, DispatcherException, \
-is_valid_email
+    is_valid_email
```

Comment:
While you're at it, use parentheses for continuations, no backslashes, please :-)